File: Python/BEGINNER/2057 Time Zone.py
# SUBPROGRAM
def calculateArrivalTime(departure, travelTime, timeZone):
    if (departure + travelTime + timeZone) >= 24:
        print("%d" % ((departure + travelTime + timeZone) - 24))
    elif (departure + travelTime + timeZone) < 0:
        print("%d" % ((departure + travelTime + timeZone) + 24))
    else:
        print("%d" % (departure + travelTime + timeZone))
    return None

# MAIN PROGRAM
departure, travelTime, timeZone = map(int, input().split())
calculateArrivalTime(departure, travelTime, timeZone)

File: C/BEGINNER/2057 Time Zone.c
#include <stdio.h>

void calculateArrival(int departure, int travelTime, int timeZone);

int main() {
    int departure, travelTime, timeZone;
    scanf("%d %d %d", &departure, &travelTime, &timeZone);
    calculateArrival(departure, travelTime, timeZone);
    return 0;
}

void calculateArrival(int departure, int travelTime, int timeZone)
{
    if ((departure + travelTime + timeZone) >= 24)
        printf("%d\n", ((departure + travelTime + timeZone) - 24));
    else if ((departure + travelTime + timeZone) < 0)
        printf("%d\n", ((departure + travelTime + timeZone) + 24));
    else
        printf("%d\n", (departure + travelTime + timeZone));
}
